openspec/changes/archive/2026-05-23-add-api-2-5-2-6-support/design.md

```
@@ -0,0 +1,128 @@
+## Context
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)